scripts/infra

```diff
@@ -61,13 +61,15 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
                 terraform remote push
                 ;;
             plan)
+                echo "Updating AWS Batch job definitions"
                 pushd "../aws-batch"
                 # TODO: Only trigger this if the pfb-analysis container has changed, otherwise skip
                 #       and reuse old version
                 # TODO: This is a bit awkward. If we `plan` but never `apply`, we'll have created
                 #       an orphaned job definition that never gets used.
-                echo "Updating AWS Batch job definitions"
-                BATCH_JOB_NAME_REVISION=$(python update_job_definition.py pfb-analysis-run-job.json "${PFB_AWS_ECR_ENDPOINT}/pfb-analysis:${GIT_COMMIT}")
+                docker-compose build
```

Comment:
So I explicitly kept this separate for two reasons:

1. It's a custom command only used in CI, so it keeps the main docker-compose.yml clean and allows us to continue using `docker-compose up` without explicitly choosing services to start
2. It allows us to simply delete this directory (`./deployment/aws-batch/`) once Terraform supports Batch without having to delete additional dependencies sprinkled throughout the project.

With that said, I definitely see the argument for consistency. Happy to move to the main `docker-compose.yml` if that's more compelling.